pyrep/robots/mobiles/mobile_base.py

```diff
@@ -94,7 +100,7 @@ def set_cartesian_position(self, position: List[float]):
 
         :param position: length 3 list containing x and y position, and angle position
 
-        NOTE: not supported for two wheel robot yet.
+        NOTE: not supported for nonholonomic mobile bases.
```

Comment:
Why do we have this limitation?

Reply:
It requires some constraints for nonholonomic robots. Haven't really thought about it yet. Could implemented it later on.